util/image.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 # tmpl.sh = /usr/local/metaserver.conf /tmp/metaserver.conf
+# $1: bs/fs $2: tag $3: os
 function tmpl() {
```

Comment:
add  `set -e` to terminate any command execution error.

Reply:
Yeah that's an easier way. But I missed it and have done it referring to utils/build.sh :sweat_smile:

Reply:
It's ok :)